pkg/collector/spec.go

```diff
@@ -444,6 +444,10 @@ func buildApiRequirements(allServiceRequirements []*ServiceRequirements, project
 						projectErrors.Add(fmt.Errorf("service %s attempted to register an OIDC security scheme with an empty issuer", serviceRequirements.serviceName))
 					}
 
+					if !strings.Contains(issuerUrl, "/.well-known/openid-configuration") {
+						projectErrors.Add(fmt.Errorf("service %s attempted to register an OIDC security policy with an invalid OIDC URL", serviceRequirements.serviceName))
+					}
```

Comment:
can we confirm that all providers conform to this as a standard? I think I remember AWS cognito being a bit different.

It might actually be worth trying to actually fetch the document and seeing if its a valid OIDC public doc.